### PR TITLE
Zuora

### PR DESCRIPTION
**Pull request by Tiseanu**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--helix-poc--enake.hlx.page/consumer/nl/new/cl-offer-opt
- After: https://header-block-update--helix-poc--enake.hlx.page/consumer/nl/new/cl-offer-opt

### CODE REVIEW

**Review by enake — APPROVED** [review]

File: scripts/zuora.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+export default class initZuoraNL {
```
Comment (vladalexandruradulescu) [inline scripts/zuora.js:1]: class names should start with capital letter and shoudn't have method name => ZuoraNLClass / ZuoraNL

File: blocks/c-productswithvpn/c-productswithvpn.css [hunk 2]
```diff
@@ -61,6 +61,7 @@
 .c-productswithvpn > div div.prices_box span.prod-oldprice {
   margin-top: 10px;
   font-size: 24px;
+  line-height: 24px;
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn/c-productswithvpn.css:64]: this should be added in .scss first

File: blocks/c-productswithvpn2/c-productswithvpn2.css [hunk 3]
```diff
@@ -173,6 +173,10 @@
   display: block;
 }
 
+.c-productswithvpn2 > div table {
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn2/c-productswithvpn2.css:176]: styles should be added in .scss first

File: styles/styles.css [hunk 4]
```diff
@@ -386,6 +386,9 @@ main .section.highlight {
   margin: 2px 0;
 }
 
+.prod-percent {
```
Comment (vladalexandruradulescu) [inline styles/styles.css:389]: styles should be added in .scss first

File: scripts/utils.js [hunk 5]
```diff
@@ -108,3 +112,280 @@ export function adobeMcAppendVisitorId(selector) {
...
+        });
+      }
+    } else {
+      allBuyLinkBox.forEach((item) => {
+        item.href = buyLink;
+      });
+    }
+  }
+
+  document.querySelectorAll(`.${productId}_box`).forEach((item) => {
```
Comment (vladalexandruradulescu) [inline scripts/utils.js:379]: this logic can be extracted here, as we discussed yesterday
image

File: blocks/c-productswithvpn/c-productswithvpn.js [hunk 6]
```diff
@@ -93,7 +94,7 @@ export default function decorate(block) {
       const prodName = productAliases(productsAsList[idx].split('/')[0]);
 
       const pricesDiv = document.createElement('div');
-      pricesDiv.className = 'prices_box';
+      pricesDiv.classList.add('prices_box', 'awaitLoader');
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn/c-productswithvpn.js:97]: awaitLoader logic shouldn't be here, move it here instead 
image

**Review by vladalexandruradulescu — CHANGES_REQUESTED** [review]

**Pull request merged**